fixup-mountpoints

```diff
@@ -533,343 +44,11 @@ case "$DEVICE" in
-            -e 's block/bootdevice/by-name/pmic sde9 ' \
-            -e 's block/bootdevice/by-name/pmicbak sde10 ' \
-            -e 's block/bootdevice/by-name/recovery sde21 ' \
-            -e 's block/bootdevice/by-name/reserve sdd1 ' \
-            -e 's block/bootdevice/by-name/reserve1 sda12 ' \
-            -e 's block/bootdevice/by-name/reserve2 sda13 ' \
-            -e 's block/bootdevice/by-name/rpm sde1 ' \
-            -e 's block/bootdevice/by-name/rpmbak sde2 ' \
-            -e 's block/bootdevice/by-name/sec sde8 ' \
-            -e 's block/bootdevice/by-name/splash sde33 ' \
-            -e 's block/bootdevice/by-name/ssd sda1 ' \
-            -e 's block/bootdevice/by-name/sti sde34 ' \
-            -e 's block/bootdevice/by-name/system sde20 ' \
-            -e 's block/bootdevice/by-name/tz sde3 ' \
-            -e 's block/bootdevice/by-name/tzbak sde4 ' \
-            -e 's block/bootdevice/by-name/userdata sda15 ' \
-            -e 's block/bootdevice/by-name/xbl sdb1 ' \
-            -e 's block/bootdevice/by-name/xblbak sdc1 ' \
-            "$@"
-        ;;
-
-    "addison")
-        sed -i \
-            -e 's block/bootdevice/by-name/boot mmcblk0p37 ' \
-            -e 's block/bootdevice/by-name/cache mmcblk0p52 ' \
-            -e 's block/bootdevice/by-name/dsp mmcblk0p22 ' \
-            -e 's block/bootdevice/by-name/fsg mmcblk0p29 ' \
-            -e 's block/bootdevice/by-name/modem mmcblk0p19 ' \
-            -e 's block/bootdevice/by-name/persist mmcblk0p30 ' \
-            -e 's block/bootdevice/by-name/recovery mmcblk0p38 ' \
-            -e 's block/bootdevice/by-name/system mmcblk0p53 ' \
-            -e 's block/bootdevice/by-name/userdata mmcblk0p54 ' \
-            "$@"
-        ;;
-
-    "mido")
-        sed -i \
-            -e 's block/bootdevice/by-name/boot mmcblk0p21 ' \
-            -e 's block/bootdevice/by-name/cache mmcblk0p25 ' \
-            -e 's block/bootdevice/by-name/config mmcblk0p29 ' \
-            -e 's block/bootdevice/by-name/dsp mmcblk0p12 ' \
-            -e 's block/bootdevice/by-name/fsc mmcblk0p2 ' \
-            -e 's block/bootdevice/by-name/misc mmcblk0p27 ' \
-            -e 's block/bootdevice/by-name/modem mmcblk0p1 ' \
-            -e 's block/bootdevice/by-name/modemst1 mmcblk0p13 ' \
-            -e 's block/bootdevice/by-name/modemst2 mmcblk0p14 ' \
-            -e 's block/bootdevice/by-name/persist mmcblk0p26 ' \
-            -e 's block/bootdevice/by-name/recovery mmcblk0p22 ' \
-            -e 's block/bootdevice/by-name/rpm mmcblk0p6 ' \
-            -e 's block/bootdevice/by-name/sbl1 mmcblk0p4 ' \
-            -e 's block/bootdevice/by-name/splash mmcblk0p18 ' \
-            -e 's block/bootdevice/by-name/ssd mmcblk0p3 ' \
-            -e 's block/bootdevice/by-name/system mmcblk0p24 ' \
-            -e 's block/bootdevice/by-name/userdata mmcblk0p49 ' \
-            -e 's block/bootdevice/by-name/oem mmcblk0p30 ' \
```

Comment:
Please do not delete devices that you do not have, they are there for others and they do not interfere.

Please avoid creating a pull request from your master branch, as I think this commit was not meant for this PR.

Reply:
Has been re-fixed.